### PR TITLE
Portable apps

### PR DESCRIPTION
**Pull request by russellhadley**
Moves all jit utils to a flat directory of portable apps with script wrappers for invocation.

Portable apps are DLLs that have only MSIL and are invoked with 'dotnet.exe'.  Moving portable apps allows us to share the dotnet framework and only include a single directory on the path but requires wrapper scripts to hide the dotnet <mytool>.dll <args> command line.

### CODE REVIEW

**Comment by russellhadley** [comment]
@RussKeldorph

**Comment by RussKeldorph** [comment]
> 

Did you consider giving this file a generic name in a central location and copying it into its final name in build.cmd/.sh?  It's not a horrible amount of duplication, but I'm not sure I see the need to have a custom one for every project, esp since they are all currently identical.

---
Refers to: src/cijobs/cijobs.cmd:3 in 2c83c61. [](commit_id = 2c83c619c60b7dd051055b4ce8717dee8dc35fec, deletion_comment = False)

File: src/jit-diff/jit-diff.cs [hunk 1]
```diff
@@ -551,15 +551,24 @@ public static int Main(string[] args)
...
+        public static CommandResult TryCommand (string name, IEnumerable<string> commandArgs, bool capture = false)
         {
             try 
             {
                 Command command =  Command.Create(name, commandArgs);
 
-                // Wireup stdout/stderr so we can see outout.
-                command.ForwardStdOut();
-                command.ForwardStdErr();
+                if (capture)
```
Comment (RussKeldorph) [inline src/jit-diff/jit-diff.cs]: Is this diff related to the other changes?  If not, should you at least comment on it in the description?

**Comment by RussKeldorph** [comment]
:shipit:

**Comment by RussKeldorph** [comment]
:+1:
